spec/resty/resolver_spec.lua

```diff
@@ -172,9 +172,9 @@ describe('resty.resolver', function()
 #search updated  in comentary
 #search nameserver 1.2.3.4
 #search nameserver
-search localdomain.example.com local
-nameserver 127.0.0.2
-nameserver 127.0.0.1
+search localdomain.example.com local #search nameserver
```

Comment:
I would have created a new test for this instead of modifying the existing one.

I think that having `it('ignores comments after search or nameserver', function() ...)` would make the tests cleaner. What do you think?

Reply:
Ok

Reply:
done a339d4bbe25e116686389a4a1c7d77b3f0d111e8